core/templates/pages/story-editor-page/services/story-editor-state.service.spec.ts

```diff
@@ -544,6 +544,21 @@ describe('Story editor state service', () => {
     expect(storyEditorStateService.areAnyExpIdsChanged()).toBeFalse();
   });
 
+  it('should set _currentNodeIsPublishable when setCurrentNodeAsPublishable ' +
+  'is called', () => {
```

Comment:
Prefer "should set current node as publishable"


Note that frontend test descriptions should not refer to internal variables or deal with implementation specifics. These can change. The tests and their descriptions should instead focus on the functionality instead. See wiki: https://github.com/oppia/oppia/wiki/Frontend-tests#test-the-interface-not-the-implementation

Reply:
Ditto below.

Reply:
Done